operators/lazy-vs-eager-execution.md

```diff
@@ -0,0 +1,44 @@
+# Lazy vs Eager Execution
+
+The default mode of operator execution is `eager`. This means that whenever an operator is called, it will immediately run. This allows us to provide useful features like type enforcement.
+
+Alternatively, you may execute an operator in `lazy` mode. This means that we will only run that operator once the entire flow is published. This removes any pre-publish guardrails 
+that `eager` mode provides, but can greatly speed up flow creation process, since we no longer run the flow before publishing it.
```

Comment:
Slight edit: "For workflows processing a large amount of data, this can greatly sleep up the workflow creation process, since..."